packages/repo-tools/src/commands/api-reports/api-reports.ts

```diff
@@ -99,22 +87,71 @@ export default async (paths: string[], opts: OptionValues) => {
       outputDir: tmpDir,
       isLocalBuild: !isCiBuild,
       tsconfigFilePath,
+      allowWarnings,
+      omitMessages: Array.isArray(omitMessages) ? omitMessages : [],
     });
   }
   if (cliPackageDirs.length > 0) {
     console.log('# Generating package CLI reports');
     await runCliExtraction({
-      projectRoot,
       packageDirs: cliPackageDirs,
       isLocalBuild: !isCiBuild,
     });
   }
 
+  console.log(isDocsBuild);
   if (isDocsBuild) {
     console.log('# Generating package documentation');
     await buildDocs({
       inputDir: tmpDir,
-      outputDir: resolvePath(projectRoot, 'docs/reference'),
+      outputDir: cliPaths.resolveTargetRoot('docs/reference'),
     });
   }
 };
+
+/**
+ * Retrieves the list of package names in the "workspaces" field of the `package.json` file in the current workspace root.
+ *
+ * If the file does not exist, or the "workspaces" field is not present, returns `undefined`.
+ *
+ * @returns {Promise<string[] | undefined>} The list of package names, or `undefined` if not found.
+ */
+async function getWorkspacePkgs() {
```

Comment:
```suggestion
async function getWorkspacePackagePathPatterns() {
```

No need to keep this short x)